orangecontrib/educational/widgets/owpolynomialclassification.py

```diff
@@ -124,7 +124,7 @@ class OWPolynomialClassification(OWBaseLearner):
 
     class Warning(OWWidget.Warning):
         to_few_features = Msg("Too few Continuous feature. Min 2 required")
```

Comment:
```
        to_few_features = Msg(
            "Polynomial classification requires at least two numeric features")
```

Besides, I guess this should be an error, not a warning. Warning informs the user that something was not done as intended, or that something extra (e.g. discretization) happened in the background, while in this case the widget "failed", hence an error.